Game.cpp

```diff
@@ -0,0 +1,146 @@
+#include "Game.h"
+#include "Map.h"
+
+void Game::setMap(Map map)
+{
+    
+}
+void Game::putHero(Hero hero,int x,int y)

```

Comment:
ebből hiányoznak a következő vizsgálatok:
- van-e map beállítva
- van-e hős már felrakva a pályára
- elindult-e már a játék

Reply:
done